src/components/UI/card/Card.tsx

```diff
@@ -2,26 +2,49 @@ import style from './Card.module.scss';
 import type { CardInfo } from '@/types';
 import { useNavigate, useSearchParams } from 'react-router';
 import { type MouseEvent } from 'react';
+import Checkbox from '../checkbox/Checkbox';
+import useSelectedItems from '@/store/selectedItems';
 
 export type CardProps = {
   info: CardInfo;
 };
 
-function Card({ info: { id, image, name } }: CardProps) {
+function Card({ info }: CardProps) {
   const navigate = useNavigate();
   const [searchParams] = useSearchParams();
-  const { card, card__img, card__title } = style;
+  const items = useSelectedItems((state) => state.items);
```

Comment:
It is better to use selector here and everywhere. You can read about it https://habr.com/ru/articles/564004/.
It should be smth like that
```suggestion
  const items = useSelectedItems(selectedItems);
```

and in the store you should add 
```
export const selectedItems = (state) => state.items;
```

Reply:
Thanks